app/src/main/res/values-fa/strings.xml

```diff
@@ -20,7 +20,7 @@
+  <string name="about_content"><![CDATA[<p xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2"><div dir="rtl">%1$s &#1588;&#1605;&#1575; &#1585;&#1575; &#1583;&#1585; &#1705;&#1606;&#1578;&#1585;&#1604; &#1602;&#1585;&#1575;&#1585; &#1605;&#1740;&#1583;&#1607;&#1583;.</p>\n<p xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">"&#1575;&#1586; &#1575;&#1740;&#1606; &#1576;&#1585;&#1606;&#1575;&#1605;&#1607; &#1576;&#1607; &#1593;&#1606;&#1608;&#1575;&#1606; &#1740;&#1705; &#1605;&#1585;&#1608;&#1585;&#1711;&#1585; &#1606;&#1575;&#1588;&#1606;&#1575;&#1587; &#1575;&#1587;&#1578;&#1601;&#1575;&#1583;&#1607; &#1705;&#1606;&#1740;&#1583;:\n  "<ul>"\n    "<li>&#1576;&#1583;&#1608;&#1606; &#1582;&#1585;&#1608;&#1580; &#1575;&#1586; &#1576;&#1585;&#1606;&#1575;&#1605;&#1607;&#1548; &#1575;&#1740;&#1606;&#1578;&#1585;&#1606;&#1578; &#1585;&#1575; &#1580;&#1587;&#1578;&#8204;&#1608;&#1580;&#1608; &#1608; &#1605;&#1585;&#1608;&#1585; &#1705;&#1606;&#1740;&#1583;</li>"\n    "<li>&#1585;&#1583;&#1740;&#1575;&#1576;&#8204;&#1607;&#1575; &#1585;&#1575; &#1605;&#1587;&#1583;&#1608;&#1583; &#1705;&#1606;&#1740;&#1583; (&#1740;&#1575; &#1575;&#1740;&#1606;&#1705;&#1607; &#1578;&#1606;&#1592;&#1740;&#1605;&#1575;&#1578; &#1582;&#1608;&#1583; &#1585;&#1575; &#1576;&#1585;&#1575;&#1740; &#1575;&#1580;&#1575;&#1586;&#1607; &#1583;&#1575;&#1583;&#1606; &#1576;&#1607; &#1585;&#1583;&#1740;&#1575;&#1576;&#8204;&#1607;&#1575; &#1578;&#1594;&#1740;&#1740;&#1585; &#1583;&#1607;&#1740;&#1583;)</li>"\n    "<li>&#1662;&#1575;&#1705; &#1705;&#1606;&#1740;&#1583; &#1578;&#1575; &#1575;&#1591;&#1604;&#1575;&#1593;&#1575;&#1578;&#1740; &#1605;&#1575;&#1606;&#1606;&#1583; &#1705;&#1608;&#1705;&#1740;&#8204;&#1607;&#1575; &#1608; &#1607;&#1605;&#1670;&#1606;&#1740;&#1606; &#1580;&#1587;&#1578;&#8204;&#1608;&#1580;&#1608;&#1607;&#1575; &#1608; &#1578;&#1575;&#1585;&#1740;&#1582;&#1670;&#1607; &#1605;&#1585;&#1608;&#1585; &#1581;&#1584;&#1601; &#1588;&#1608;&#1606;&#1583;</li>"\n  "</ul>\n</p>\n<p xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">%1$s &#1578;&#1608;&#1587;&#1617;&#1591; &#1605;&#1608;&#1586;&#1740;&#1604;&#1575; &#1605;&#1606;&#1578;&#1588;&#1585; &#1588;&#1583;&#1607; &#1575;&#1587;&#1578;. &#1605;&#1571;&#1605;&#1608;&#1585;&#1740;&#1578; &#1605;&#1575; &#1578;&#1575; &#1585;&#1608;&#1575;&#1580; &#1740;&#1705; &#1575;&#1740;&#1606;&#1578;&#1585;&#1606;&#1578; &#1587;&#1575;&#1604;&#1605;&#1548; &#1576;&#1575;&#1586; &#1575;&#1587;&#1578;.<br/>\n<a href="%2$s">&#1576;&#1740;&#1588;&#1578;&#1585; &#1576;&#1583;&#1575;&#1606;&#1740;&#1583;</a></div></p>]]></string>
```

Comment:
Making this change in the string files is problematic: They are automatically imported from the L10N repo and translators use a bunch of different tools to translate them. Even if we convince those translators to update the strings then this problem might come back as soon as we add other RTL languages.

Initially I made this content RTL compatible by adding `dir=auto` to the body tag here:
https://github.com/mozilla-mobile/focus-android/blob/master/app/src/main/res/raw/about.html#L10

I wonder if we could instead insert ltr/rtl based on the current configuration (like we insert other strings into this HTML content). It seems like "auto" doesn't work for all our supported locales. :(